### ISSUE TITLE
 Error: In Music Separation - Unable to read the model: htdemucs_v4.xml

### ISSUE THREAD

**Issue by NevGKA** [intel/openvino-plugins-ai-audacity#151]
Hey there,

New to Audacity and trying to get my head around removing vocals from an mp3 file.

OS Name	Microsoft Windows 10 Home
Version	10.0.19045 Build 19045
System Type	x64-based PC
Processor	Intel(R) Core(TM) i3-7100 CPU @ 3.90GHz, 3912 Mhz, 2 Core(s), 4 Logical Processor(s)

I have to confess I found the instructions confusing (_the written instructions evidently assume a level of experience and familiarity with this sort of thing that I don't posses, and information in the video no longer appears to me to to be in keeping with packages to be downloaded?   I feel like I might have missed a step but not sure what it is?_). 

Have managed to install and enable `open-vino` but getting errors:

```
20:13:22: Audacity 3.5.1
20:13:22: Error: Failed to load shared library 'avformat-60.dll' (error 126: The specified module could not be found.)
20:13:22: Error: Failed to load shared library 'avformat-59.dll' (error 126: The specified module could not be found.)
20:13:22: Error: Failed to load shared library 'avformat-58.dll' (error 126: The specified module could not be found.)
20:13:22: Error: Failed to load shared library 'avformat-57.dll' (error 126: The specified module could not be found.)
20:13:22: Error: Failed to load shared library 'avformat-55.dll' (error 126: The specified module could not be found.)
20:13:25: Error: Failed to load shared library 'avformat-60.dll' (error 126: The specified module could not be found.)
20:13:25: Error: Failed to load shared library 'avformat-59.dll' (error 126: The specified module could not be found.)
20:13:25: Error: Failed to load shared library 'avformat-58.dll' (error 126: The specified module could not be found.)
20:13:25: Error: Failed to load shared library 'avformat-57.dll' (error 126: The specified module could not be found.)
20:13:25: Error: Failed to load shared library 'avformat-55.dll' (error 126: The specified module could not be found.)
20:13:38: File name is C:\Users\user\Downloads\test.mp3
20:13:38: Mime type is *
20:13:38: Opening with libmpg123
20:13:38: Open(C:\Users\user\Downloads\test.mp3) succeeded
20:13:40: Operation 'Importing MP3 files' took 2.113000 seconds. Poll was called 16601 times and took 0.393279 seconds. Yield was called 15 times and took 0.008403 seconds.
20:13:42: Operation 'Music Information Retrieval' took 0.000000 seconds. Poll was called 0 times and took 0.000000 seconds. Yield was called 0 times and took 0.000000 seconds.
20:13:58: Operation 'Pre-processing' took 0.000000 seconds. Poll was called 0 times and took 0.000000 seconds. Yield was called 0 times and took 0.000000 seconds.
20:13:58: Error: In Music Separation, exception: Exception from src\inference\src\cpp\core.cpp:126:
Exception from src\inference\src\model_reader.cpp:160:
Unable to read the model: C:\Program Files\Audacity\openvino-models\htdemucs_v4.xml Please check that model format: xml is supported and the model is correct. Available frontends: ir onnx paddle pytorch tf tflite 


20:14:00: Operation 'OpenVINO Music Separation' took 0.132000 seconds. Poll was called 2 times and took 0.000002 seconds. Yield was called 0 times and took 0.000000 seconds.
```
`"....\openvino-models\htdemucs_v4.xml"` file does not exist in the folder?

![image](https://github.com/intel/openvino-plugins-ai-audacity/assets/168345920/c03ccae4-b2e8-4a04-9af9-0a59ee31456b)

Any assistance would be greatly appreciated?

Cheers!

**Comment by RyanMetcalfeInt8** [comment]
Hi @NevGKA, which version of Audacity are you using, and which version of these plugins did you install?

From your log, it appears that it's using one of these recent plugin releases (3.5.0-R2, or 3.5.1-R2), but the screenshot of your openvino-models folder shows contents of ```openvino-models.zip``` from our original ```3.4.2-R1``` release.

> I have to confess I found the instructions confusing (the written instructions evidently assume a level of experience and familiarity with this sort of thing that I don't posses, and information in the video no longer appears to me to to be in keeping with packages to be downloaded? I feel like I might have missed a step but not sure what it is?).

I think you're referring to instructions for 3.4.2-R1 release? Anyway, installation for latest release (https://github.com/intel/openvino-plugins-ai-audacity/releases/tag/v3.5.1-R2) should be much easier, as everything is wrapped into an installer. Basically, run the installer, select the features (models) that you want to install, and the installer should put everything into the right place. Just make sure you install the latest version of Audacity (3.5.1).

Thanks,
Ryan

**Comment by NevGKA** [comment]
DOH! Classic PICINC issue. Thanks Ryan,

**Comment by DJ5D** [comment]
I encountered this same issue, but to make it work there was one extra step:

When applying the hotfix, it advises: 

"At the Select Models step, choose Install no models from the drop-down list. This way, you won't need to re-download all of the models again"

Ignore this advice. The first time I followed those instructions "htdemucs_v4.xml" was NOT installed.

I applied the hotfix again and chose "install all models"

NOW it works fine.
